app/views/subscriptions/_card.html.erb

```diff
@@ -49,7 +49,7 @@
         </span>
       </p>
 
-      <div class="card-text mb-3">
+      <div class="card-text mb-3" dir='ltr'>
```

Comment:
You may also need to add the `ltr` _class_ to the element (otherwise the `rtl` on an ancestor element will take precedence and _could_ override certain styles within this div).